### PR TITLE
feat: [IOPID-1500] Adopt the new DS to `IngressScreen`

### PR DESCRIPTION
**Pull request by shadowsheep1**
## Short description
This PR addresses the redesign of the `IngressScreen` by adopting the new DS. 

> [!TIP]
> Review Privacy ✅
> [MP event](https://pagopa.atlassian.net/wiki/spaces/CM1/pages/873758721/TPR+Identificazione+PIN+BIO) ✅
> Translations ✅
>
> [Run e2e tests](https://github.com/pagopa/io-app/actions/workflows/test-e2e.yml)

## List of changes proposed in this pull request
- Adopt of the new DS to `IngressScreen`
- Fix the EIC UAT selection on `LandingScreen`. This is a regression from [here](https://github.com/pagopa/io-app/pull/5536).
- Add missing English translations from [here](https://github.com/pagopa/io-app/pull/5536).
- Remove unused components.
- Fix a glitch present in `LandingScreen` on some iOS devices.
- Prefer on Android an **announcement** instead of setting the focus on the screen content title because the screen is show just a short period of time. On **iOS** if the screen is visible for a long time, we fallback to the standard accessibility behavior.

<details><summary>🎦 Demo 🎦</summary>
<p>

| Android a11y `IngressScreen` | iOS a11y `IngressScreen` | 
| - | - | 
| <video src="https://github.com/pagopa/io-app/assets/16268789/9b4d96f0-3ae8-4e42-9246-7a4d22d183a3" /> | <video src="https://github.com/pagopa/io-app/assets/16268789/6629280d-803f-466a-88c4-fabff612cf9a" /> |

| iOS glitch ❌ | iOS Fixed ✅ |
| - | - | 
| <video src="https://github.com/pagopa/io-app/assets/16268789/993663ce-1696-42ea-8883-d24d5336a31c" /> | <video src="https://github.com/pagopa/io-app/assets/16268789/65577da2-c68f-4841-a5df-3eedf0aa12cb" /> | 

</p>
</details> 

## How to test
- Test a login flow on both Android and iOS devices.
- Test a11y on both Android and iOS devices (**TIP** restart the app after a successful login and stop the dev server: the startup saga'll stop on the `IngressScreen`).
- Check the [MP event](https://pagopa.atlassian.net/wiki/spaces/CM1/pages/873758721/TPR+Identificazione+PIN+BIO).
- Test EIC UAT selection flow.
- Test the fix on the glitch in `LandingScreen` (**TIP** do a standard login — no LV — and stop and restart the dev server, kill the app and reopen it, you should see no glitch and the session expired pictogram).

### CODE REVIEW

**Comment by shadowsheep1** [comment]
@mariateresaventura copy review needed! Please take care also this English keys we forgot [here ](https://github.com/pagopa/io-app/pull/5536/files#diff-1f1454493070fa76fae9590191905cdfc0d8d9fc724ed1bb0de35027f71fe372).

File: ts/screens/ingress/IngressScreen.tsx [hunk 1]
```diff
@@ -1,104 +1,63 @@
...
-          </List>
-          <View style={{ marginTop: 48 }}>
-            <SectionStatusComponent sectionKey={"ingress"} />
-          </View>
-        </Container>
-      </SafeAreaView>
-    );
-  }
-}
+const SPINNER_SIZE = 48;
```
Comment (francescopersico) [inline ts/screens/ingress/IngressScreen.tsx:26]: Is this value only used for this screen or is a generic rule so we can extract in a global definition?

File: ts/screens/ingress/IngressScreen.tsx [hunk 2]
```diff
@@ -1,104 +1,63 @@
...
+const SPACE_BETWEEN_SPINNER_AND_TEXT = 24;
```
Comment (francescopersico) [inline ts/screens/ingress/IngressScreen.tsx:27]: As above.

File: ts/screens/ingress/IngressScreen.tsx [hunk 3]
```diff
@@ -1,104 +1,63 @@
...
+      // and the focus is moved to another element.
+      AccessibilityInfo.announceForAccessibility(contentTitle);
+    }
+  });
 
-export default connect(mapStateToProps)(IngressScreen);
+  return (
+    <SafeAreaView style={styles.container}>
+      <ContentWrapper>
+        <View style={{ alignItems: "center" }}>
```
Comment (francescopersico) [inline ts/screens/ingress/IngressScreen.tsx:47]: No inline style?

File: ts/screens/ingress/IngressScreen.tsx [hunk 4]
```diff
@@ -1,104 +1,63 @@
...
+          <View
+            accessible={false}
+            accessibilityElementsHidden={true}
+            importantForAccessibility={"no-hide-descendants"}
+          >
+            <LoadingSpinner size={SPINNER_SIZE} />
+          </View>
+          <VSpacer size={SPACE_BETWEEN_SPINNER_AND_TEXT} />
+          <H3 style={{ textAlign: "center" }} accessibilityLabel={contentTitle}>
```
Comment (francescopersico) [inline ts/screens/ingress/IngressScreen.tsx:56]: No inline style?

**Review by LazyAfternoons — COMMENTED** [review]
Working as expected.
The CIE UAT flow is working. Should we update our internal documentation on how to use it since it's 3 taps on the dots now?. 
MP event are properly tracked and no glitches whatsoever with the `SafeArea`, except for this one but I'm usure if it's related as it happens on `master` too: 

https://github.com/pagopa/io-app/assets/12371664/d52084fe-58e0-4009-9446-464c1a0d36a3


We can merge this once other comments have been addressed.

File: ts/screens/ingress/IngressScreen.tsx (hunk 1 above)
Comment (shadowsheep1) [reply ts/screens/ingress/IngressScreen.tsx]: Good question. This let me be aware that the component to use is not the one from the DS directly but the `LoadingIndicator` [that wraps it with the correct styling](https://pagopaspa.slack.com/archives/C013V764P9U/p1707928510151789).

File: ts/screens/ingress/IngressScreen.tsx (hunk 3 above)
Comment (shadowsheep1) [reply ts/screens/ingress/IngressScreen.tsx]: When is only a single style value I put it directly without using an inline style. Anyway I addressed it with an inline style here f7e95f836a2f448be9a82f5019e577b11e25d30c

File: ts/screens/ingress/IngressScreen.tsx (hunk 4 above)
Comment (shadowsheep1) [reply ts/screens/ingress/IngressScreen.tsx]: As above.

**Review by francescopersico — APPROVED** [review]

File: locales/en/index.yml [hunk 5]
```diff
@@ -558,7 +558,7 @@ authentication:
         hint: Carousel that contains an introduction guide to IO functionalities. Select to know the content.
     session_expired:
       title: Your session is expired
-      body: Per ragioni di sicurezza, ti chiediamo di entrare con SPID o CIE ogni {{days}} giorni o se esci dall’app.
+      body: For security reasons, we ask you to log in with SPID or CIE every {{days}} day or if you log out
```
Comment (francescopersico) [inline locales/en/index.yml:561]: We need to start using [plurals](https://www.i18next.com/translation-function/plurals) do differentiate 1 day from 5 day**s**.
Take note.

File: locales/en/index.yml [hunk 6]
```diff
@@ -586,9 +586,9 @@ authentication:
     card2-content: You can pay a tax or a penalty by your bank account and your cards
     card3-title: Quickly pay a physical notice
     card3-content: You can pay a notice also by scanning the QR code on the physical notice    
-    card5-title: Welcome to the public version of IO!
-    card5-content: Vediamo insieme cosa puoi fare
-    card5-content-accessibility: Seleziona elemento precedente per aprire una chat con il team di IO se qualcosa non funziona come dovrebbe.
+    card5-title: Welcome to IO!
+    card5-content: What you can do?
```
Comment (francescopersico) [inline locales/en/index.yml:590]:
```suggestion
    card5-content: What can you do?
```

File: locales/en/index.yml (hunk 5 above)
Comment (shadowsheep1) [reply locales/en/index.yml:561]: Thank you! In this case days are always plural and we have a translation issue. We need to update `day` with `days`.

Comment (shadowsheep1) [reply locales/en/index.yml]: Fixed! Thank you! 4e6cc4f29a8acd59d0b6627f45137bc3a664d4c4

**Pull request merged**